packages/inspection-report/src/components/Gallery.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useState } from 'react';
+import { Modal, StyleSheet, View, Image, Pressable, Text } from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+
+import Thumbnail from './common/Thumbnail';
+
+const styles = StyleSheet.create({
+  container: {
+    alignContent: 'flex-start',
+    flex: 1,
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    gap: 10,
+    justifyContent: 'center',
+    paddingVertical: 15,
+  },
+  closeButtonWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    color: '#fff',
+    alignItems: 'center',
+    paddingHorizontal: 15,
+    paddingVertical: 5,
+    backgroundColor: '#121212',
+  },
+  header: {
+    fontSize: 20,
+  },
+});
+
+const pictureList = [
```

Comment:
1. Create a file named `mock.js` in the `src` directory
2. Place these fake pictures in this file and export them
3. Import the pictures in the DamageReport component
4. Pass them as a prop to the Gallery component